BDArmory/Damage/HitpointTracker.cs

```diff
@@ -835,37 +836,42 @@ public float CalculateTotalHitpoints()
                         //hitpoints = (structuralVolume * Mathf.Pow(density, .333f) * Mathf.Clamp(80 - (structuralVolume / 2), 80 / 4, 80)) * hitpointMultiplier * 0.333f; //volume * cuberoot of density * HP mult scaled by size
 
                         if (isProcWing)
-                        {/*
-                            if (FerramAerospace.CheckForFAR())
+                        {
+                            if (!BDArmorySettings.RUNWAY_PROJECT) //how granular do we want to get with this...? Keep this inside the RP toggle, or spin it off into a new one?
```

Comment:
I think having this as a separate toggle and overridden by the RWP toggle would be better. There's bound to be groups outside of RWP and FJRT that want to enable this balancing.